core/src/epicli/data/azure/defaults/infrastructure/storage-share.yml

```diff
@@ -0,0 +1,9 @@
+kind: infrastructure/storage-share
+version: 0.3.0
```

Comment:
Is that correct? Other versions in this PR were bumped to 0.4.0

Reply:
Fixed